### PR TITLE
Paas 1321 rethink haproxy conf

### PR DESCRIPTION
**Pull request by adoux**
JIRA issue: https://jira.jahia.org/browse/PAAS-1321

Short description:

### CODE REVIEW

File: packages/common/restore.yml [hunk 1]
```diff
@@ -299,10 +299,9 @@ actions:
...
         env_region=$(awk -F'=' '$1=="JEL_REGION" {print $2}' /metadata_from_HOST | sed 's/[[:punct:]]//g'); export env_region
         cd jelastic_backup
-        if python3 backrest.py -a download --backupname ${settings.backup_name} --bucketname ${globals.bucketname} -m ${settings.backtype} -t ${settings.timestamp} -f haproxy.tar.gz 2>>${globals.logsPath}; then
-          ls haproxy.tar.gz || exit 0
-        fi
-        python3 backrest.py -a download --backupname ${settings.backup_name} --bucketname ${globals.bucketname} -m ${settings.backtype} -t ${settings.timestamp} -f haproxy-00-global.cfg.gz 2>>${globals.logsPath} || exit 1
+        python3 backrest.py -a download --backupname ${settings.backup_name} --bucketname ${globals.bucketname} -m ${settings.backtype} -t ${settings.timestamp} -f haproxy.tar.gz 2>>${globals.logsPath}
+        [ -f haproxy.tar.gz ] && exit 0
+        python3 backrest.py -a download --backupname ${settings.backup_name} --bucketname ${globals.bucketname} -m ${settings.backtype} -t ${settings.timestamp} -f haproxy-00-global.cfg.gz 2>>${globals.logsPath}
```
Comment (Ronan-Gorain) [inline packages/common/restore.yml:304]: I think the || exit 1 was mandatory here to support (very, very) old backups that have no haproxy conf. It's been so long that we could get rid of this backward compatibility, but then we have rework the next if so to exit with error if errOut is not  empty

File: packages/jahia/migrations/migrate-to-v7.yaml [hunk 2]
```diff
@@ -178,31 +178,21 @@ actions:
...
-          echo $(awk '/user .* password .*/ {print $2" "$4}' $OLD_HAPROXY_CONF)
+        HAPROXY_DIR=/etc/haproxy
+        OLD_HAPROXY_CONF=$HAPROXY_DIR/haproxy.cfg.d/00-global.cfg
+        HAPROXY_CONF=$HAPROXY_DIR/haproxy.cfg.jahia/jahia-cloud.cfg
+        user=$(awk '/user .* password .*/ {print $2}' $OLD_HAPROXY_CONF)
+        password=$(awk '/user .* password .*/ {print $4}' $OLD_HAPROXY_CONF)
+        sed -i "s;^\(\s*user \).*\( password \).*$;\1$user\2$password;" $HAPROXY_CONF
+        old_auth_basic_disabled=$(grep -c  "#acl tools.*#HTTP_AUTH_BASIC" $OLD_HAPROXY_CONF)
+        auth_basic_disabled=$(grep -c  "#acl tools.*#HTTP_AUTH_BASIC" $HAPROXY_CONF)
+        [ $old_auth_basic_disabled -eq $auth_basic_disabled ] && echo OUI
```
Comment (Ronan-Gorain) [inline packages/jahia/migrations/migrate-to-v7.yaml:189]: && echo OUI ? :)

**Review by Ronan-Gorain — CHANGES_REQUESTED** [review]

File: packages/common/restore.yml (hunk 1 above)
Comment (kevinvandel) [reply packages/common/restore.yml:304]: > It's been so long that we could get rid of this backward compatibility

I think we can get rid of it indeed.

File: packages/jahia/migrations/migrate-to-v7.yaml (hunk 2 above)
Comment (adoux) [reply packages/jahia/migrations/migrate-to-v7.yaml]: :facepalm: :facepalm: 
Fixed

File: packages/common/restore.yml (hunk 1 above)
Comment (adoux) [reply packages/common/restore.yml]: I added the `|| exit 1` again

**Review by kevinvandel — APPROVED** [review]

File: packages/jahia/migrations/migrate-to-v7.yaml (hunk 2 above)
Comment (kevinvandel) [reply packages/jahia/migrations/migrate-to-v7.yaml]: echo NON :trollface:

**Pull request merged**